doc/data/messages/u/use-implicit-booleaness-not-comparison-to-zero/good.py

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
```suggestion
```

Can we run `black` over these?

Reply:
Not on all of them but yeah, we're running ruff already. I'm going to open another MR.

Reply:
#8650